packages/mdx/test/Themed.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jsxImportSource react
+ * @jest-environment jsdom
+ */
+import { render } from '@theme-ui/test-utils'
+import { ThemeProvider } from '@theme-ui/core'
+import { matchers } from '@emotion/jest'
+
+import { Themed } from '../src'
+
+expect.extend(matchers)
+
+const _ = <Themed /> // <- TODO: Remove this API. This is totally redundant.
```

Comment:
What's this for? Not immediately seeing it being used

Reply:
Good catch!